frontend/src/pages/ApprovedActivityReport/components/ViewTable.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ViewTable.css';
+
+function renderData(data) {
+  if (Array.isArray(data)) {
+    return <ul>{data.map((line) => <li key={line}>{line}</li>)}</ul>;
+  }
+
+  // eslint-disable-next-line react/no-danger
+  return <span dangerouslySetInnerHTML={{ __html: data }} />;
```

Comment:
Yes indeed

Reply:
1) Can we display these fields the same way they are shown in the old review section (I think `frontend/src/pages/ActivityReport/pages/Review/HtmlReviewItem`)?
2) If not can we add a comment explaining why we have to set the inner html (because I will forget why and ask in the future!)

Reply:
whoa thanks for the heads up, will do